fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/multi_tokenizer.py

```diff
@@ -0,0 +1,1304 @@
+from typing import Dict
+from collections.abc import Iterable
+from fuse.utils import NDict
+from fuse.data import OpBase, get_sample_id
+from tokenizers import Tokenizer, Encoding
+import tokenizers
+from warnings import warn
+from typing import Optional, List, Set, Union, Tuple, Any, Iterator
+import json
+import transformers
+
+
+class ModularTokenizer(transformers.PreTrainedTokenizerBase):
```

Comment:
Why is it part of the example?
Can we move it to a common place? (fusedrug/data/ or bmfm-core)

Reply:
Copied to fusedrug/data. Will delete the example part once you approve the PR.